pruntime/enclave/src/contracts/diem.rs

```diff
@@ -246,6 +251,20 @@ impl contracts::Contract<Command, Request, Response> for Diem {
     fn handle_query(&mut self, origin: Option<&chain::AccountId>, req: Request) -> Response {
         let inner = || -> Result<Response, Error> {
             match req {
+                Request::SetTrustedState { trusted_state_b64 } => {
```

Comment:
For the convenience of debugging, we can allow setting `trusted_state` multiple times, but it shouldn't be the case in the production. Let's add a comment here:

```
// TODO: disable in read product
```